### PR TITLE
JBPM-5105: added defaultPackage

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by jomarko** [comment]
@bbrodt could you please have a look on this PR?

**Comment by jomarko** [comment]
@pleacu please, any chance to merge this?

**Comment by pleacu** [comment]
@jomarko - I don't have push rights to this repo - @krisv - can you assist Jozef?

File: drools-eclipse/org.drools.eclipse/src/main/resources/org/drools/eclipse/wizard/project/sample.bpmn.template [hunk 1]
```diff
@@ -11,7 +11,7 @@
              xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
              xmlns:tns="http://www.jboss.org/drools">
 
-  <process id="com.sample.bpmn.hello" tns:packageName="defaultPackage" name="Hello World" isExecutable="true" processType="Private">
+  <process id="com.sample.bpmn.hello" tns:packageName="defaultPackage" name="Hello World" isExecutable="true" processType="Private" tns:packageName="defaultPackage">
```
Comment (krisv) [inline drools-eclipse/org.drools.eclipse/src/main/resources/org/drools/eclipse/wizard/project/sample.bpmn.template]: @jomarko apologies for the late response, but isn't this adding the sane property that is already available on the same line (second place, after id)?

**Comment by jomarko** [comment]
@krisv you really seems to be right

**Pull request closed without merging**